app/src/main/java/com/android/bookswap/model/chat/FirebaseMessagingService.kt

```diff
@@ -0,0 +1,113 @@
+package com.android.bookswap.model.chat
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Intent
+import android.os.Build
+import androidx.core.app.NotificationCompat
+import com.android.bookswap.MainActivity
+import com.android.bookswap.R
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+
+/** Service to handle Firebase Cloud Messaging notifications. */
+open class MyFirebaseMessagingService : FirebaseMessagingService() {
+  /**
+   * Called when a message is received.
+   *
+   * @param remoteMessage the message received from Firebase Cloud Messaging.
+   */
+  override fun onMessageReceived(remoteMessage: RemoteMessage) {
+    val notification = remoteMessage.notification
+    val title = notification?.title ?: "Hello There!"
```

Comment:
This is hard-coded. Add a docstring to remind us to update it later when we have the string file.